inc/Engine/Media/AboveTheFold/Frontend/Controller.php

```diff
@@ -412,15 +412,15 @@ public function inject_beacon( $html, $url, $is_mobile ): string {
 		 *
 		 * @param int $delay The delay in milliseconds. Default is 500.
 		 */
-		$delay = apply_filters( 'rocket_lcp_delay', $default_delay );
```

Comment:
if we change the filter name, we should add an apply_filters_deprecated call too for backward compatibility